### PR TITLE
RRD: Add templates for asthma PDF

### PR DESCRIPTION
**Pull request by allthesignals**
Rough draft for asthma PF generation, all template-based. Includes some tests. WIP. Depends on #9523 

## Description of change
Adds a new disability-specific template for asthma (see app/services/rapid_ready_for_decision/views/asthma.erb). 

Todo: re-uses the "medications" template but this needs to be updated to reflect some differences in view.

## Original issue(s)
https://www.notion.so/Generate-asthma-PDF-10f4de575ee3463fa08265761b45bd9a

### CODE REVIEW

File: app/services/rapid_ready_for_decision/views/asthma/rating_schedule.erb [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<%#
+  Note: This templating language is HTML-inspired. It does not always behave like true HTML.
+  To see limitations, check out the documentation: https://github.com/puzzle/prawn-markup.
+%>
+
+<h4>Hypertension Rating Schedule</h4>
```
Comment (yoomlam) [inline app/services/rapid_ready_for_decision/views/asthma/rating_schedule.erb:6]:
```suggestion
<h4>Asthma Rating Schedule</h4>
```

**Review by yoomlam — COMMENTED** [review]
The added `spec/support/vcr_cassettes/.../when_pdf_is_for_hypertension/includes_the_veterans_name.yml` files should probably be moved/renamed instead of added, i.e. you need to delete the old cassettes.

**Comment by allthesignals** [comment]
> The added `spec/support/vcr_cassettes/.../when_pdf_is_for_hypertension/includes_the_veterans_name.yml` files should probably be moved/renamed instead of added, i.e. you need to delete the old cassettes.

I moved some things to shared_examples. For new examples, I just copied the cassette YAML. So these will appear in git as additions, but most of them will be renamed.

File: spec/services/rapid_ready_for_decision/fast_track_pdf_generator_spec.rb [hunk 2]
```diff
@@ -54,46 +54,60 @@
...
-      end.compact
+      it 'includes the veterans blood pressure readings' do
+        expect(subject).to include 'Blood pressure: 115/87'
+      end
 
-      expect(subject).to include(*dosages)
-    end
+      it 'includes the veterans medications' do
+        dosages = parsed_medications_data.map do |per|
+          next if per['dosageInstructions'].blank?
```
Comment (yoomlam) [inline spec/services/rapid_ready_for_decision/fast_track_pdf_generator_spec.rb:78]: I don't know what `per` means, so suggesting:
```suggestion
        dosages = parsed_medications_data.map do |med|
          next if med['dosageInstructions'].blank?
```

**Review by yoomlam — DISMISSED** [review]

**Review by yoomlam — APPROVED** [review]

**Review by yoomlam — APPROVED** [review]

File: spec/services/rapid_ready_for_decision/fast_track_pdf_generator_spec.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
   if ENV['_SAVE_RRD_PDF_FILE']
     after do |example|
-      file_name = "tmp/rrd-pdf-preview-#{example.metadata[:description].parameterize}-#{Time.now.to_i}.pdf"
+      file_name = "tmp/rrd-pdf-preview-#{example.full_description.parameterize}-#{Time.now.to_i}.pdf"
```
Comment (leikkisa) [inline spec/services/rapid_ready_for_decision/fast_track_pdf_generator_spec.rb:13]: Was wondering about full_description here but got cleared up by Matt, sounds like it's an rspec method.

**Review by leikkisa — APPROVED** [review]
🎉

**Review by thilton-oddball — APPROVED** [review]

**Pull request merged**